lib/ansible/modules/user.py

```diff
@@ -206,9 +206,23 @@
             - Implementation differs by platform. This option does not always mean the user cannot login using other methods.
             - This option does not disable the user, only lock the password.
             - This must be set to V(false) in order to unlock a currently locked password. The absence of this parameter will not unlock a password.
-            - Currently supported on Linux, FreeBSD, DragonFlyBSD, NetBSD, OpenBSD.
+            - Currently supported on Linux, FreeBSD, DragonFlyBSD, NetBSD, OpenBSD.    
```

Comment:
trailing whitespaces upsetting the linters?

Reply:
Hey @webknjaz 
I don't get it, was that supposed to be a question?

Reply:
The PR adds trailing whitespace, which is a linting error.

> The test ansible-test sanity --test pep8 [[explain](https://docs.ansible.com/ansible-core/devel/dev_guide/testing/sanity/pep8.html)] failed with 2 errors:
>
> lib/ansible/modules/user.py:209:84: W291: trailing whitespace
> lib/ansible/modules/user.py:223:21: W291: trailing whitespace

```suggestion
            - Currently supported on Linux, FreeBSD, DragonFlyBSD, NetBSD, OpenBSD.
```